src/main/java/org/apache/mesos/specification/DefaultTaskSet.java

```diff
@@ -16,6 +16,7 @@
     private final String name;
     private final List<TaskSpecification> taskSpecifications;
 
+    // TODO: this should be deprecated
```

Comment:
Indeed that is valid but it misses out on various other optional elements and I thought it might be unnecessary to strip down the `TaskInfo` so much. But I guess, let's keep it.